src/app/payments/send-intraledger.ts

```diff
@@ -225,19 +234,34 @@ const executePaymentViaIntraledger = async ({
       return new ResourceExpiredLockServiceError(signal.error?.message)
     }
 
-    const lnIntraLedgerMetadata = LedgerFacade.WalletIdIntraledgerLedgerMetadata({
-      paymentFlow,
-
-      amountDisplayCurrency: converter.fromUsdAmount(paymentFlow.usdPaymentAmount),
-      feeDisplayCurrency: 0 as DisplayCurrencyBaseAmount,
-      displayCurrency: DisplayCurrency.Usd,
-
-      memoOfPayer: memo || undefined,
-      senderUsername: senderAccount.username,
-      recipientUsername,
-    })
-    const { metadata, debitAccountAdditionalMetadata: additionalDebitMetadata } =
-      lnIntraLedgerMetadata
+    let metadata:
+      | NewAddWalletIdIntraledgerSendLedgerMetadata
+      | NewAddWalletIdTradeIntraAccountLedgerMetadata
+    let additionalDebitMetadata = {}
```

Comment:
this is typed as any

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/1682/commits/ca1f2e0d401ebddf53bd7394d67d2038584f3c87